doc/README.md

```diff
@@ -3,11 +3,25 @@
 2. Administrator dodaje członków Jury do edycji konkursu.
 3. Administrator aktywuje edycję konkursu.
 4. Administrator dodaje aplikanta z organizacji (tworzy konto), który chce wziąć udział w konkursie.
-    * Jeżeli organizacji nie ma w systemie zostaje utworzona.
-    * Jeżeli organizacja była w systemie zostaje wybrana a dane uzupełnione.
+    * Jeżeli organizacji nie ma w systemie, zostaje ona utworzona.
+    * Jeżeli organizacja była w systemie, zostaje wybrana a dane uzupełnione.
     * Max 1 osoba z organizacji.
     * Aplikant odbiera maila / aktywuje konto
 5. Aplikant rejstruje projekt.
+6. Aplikant dokonuje zmian w projekcie, do momentu upłynięcia terminu - planowanej daty dostarczenia raportu aplikacyjnego.
+7. Biuro nagrody przypisuje wysyła propozycje zostania asesorami do ekspertów PEM, dla każdego zgłoszenia projektu.
```

Comment:
One or the other

Reply:
Z wymagań: _Dla każdego z proponowanych ekspertów IPMA:
2. Biuro Nagrody kontaktuje się z proponowanym dla projektu ekspertem IPMA z propozycją współpracy_  Nie ma na tym etapie przypisania więc ja bym zostawił wysyła.